Dynamo/Dynamo.SwiftLang/SLLine.cs

```diff
@@ -5,22 +5,25 @@
 
 namespace Dynamo.SwiftLang {
 	public class SLLine : DelegatedSimpleElement, ISLStatement {
-		public SLLine (ISLExpr contents)
+		public SLLine (ISLExpr contents, bool addSemicolon = true)
 		{
 			Contents = Exceptions.ThrowOnNull (contents, nameof(contents));
-			if (!(contents is ISLLineable))
+			if (!(contents is ISLLineable) && addSemicolon)
 				throw new ArgumentException ("contents must be ILineable", nameof (contents));
```

Comment:
The exception text does not match the logic anymore

Also `ISLLineable`(code)  != `ILineable` (text)